addons/jenkins/1.x/jenkins.yaml

```diff
@@ -21,9 +21,6 @@ spec:
       enabled: true
     - name: none
       enabled: true
-  requires:
-    - matchLabels:
-        kubeaddons.mesosphere.io/provides: traefik
```

Comment:
In my own tests, I have not seen a NEED for traefik, though it's nice to have. I'm :heavy_check_mark: on removing this req.